dialect/sql/sqlgraph/graph.go

```diff
@@ -825,7 +827,7 @@ type creator struct {
 func (c *creator) node(ctx context.Context, tx dialect.ExecQuerier) error {
 	var (
 		edges  = EdgeSpecs(c.Edges).GroupRel()
-		insert = c.builder.Insert(c.Table).Default()
+		insert = c.builder.Insert(c.Table).Schema(c.Schema).Default()
```

Comment:
Let's move all changes in this file to a separate PR with tests in `graph_test.go` - ping me in Slack if you need help with this. 

Also, some parts are missing here, but I can help with this or can guide you.